packages/sdk/src/util/restify.plugins.serveStaticFiles.d.ts

```diff
@@ -0,0 +1,21 @@
+/*!
```

Comment:
custom .d.ts files should go into a `types` folder outside `src`, and then that folder is added to tsconfig.json's  `types` entry.